terms_of_reference.md

```diff
@@ -0,0 +1,219 @@
+
+The Society requests that one of the SIG committee reports on progress to the Society trustee meetings every three months. This can be the same person each time or change as necessary depending on availability. The report can be made in person, or emailed to any one of the trustees for inclusion in the agenda. If required, the Society can create a folder on the Google drive for the SIG’s documents.
+
+## Committee Roles
+
+<!-- How will the SIG committee be structured? Which roles, if any, will be taken by members of the Society? How will volunteers be supported and guided? Will there be a steering group within the committee, if so which roles will be part of it? When will roles be appointed, and what is the process? Include a diagram/image of the committee structure if needed. -->
+
+The SIG committee will consist of a majority of volunteers without any requirement that they be members of the Society. The SIG steering group will comprise a smaller section of the committee and most will be Society members. The Steering Group should ensure that working as a committee member is a fun and learning experience. To that end, we look for a proactive attitude from the Steering Group to ensure that tasks are evenly spread, giving opportunities to all volunteers to learn new skills and gain confidence. 
+
+Four volunteers will form the Steering Group:
+
+- *Chair*
+- *Deputy-Chair*
+- *Secretary*
+- *Editor*
+
+This steering group takes responsibility collectively for all decisions and can vote when unable to agree unanimously. If this vote doesn’t lead to agreement, the chair has the casting vote.
+
+For induction, the Steering Group is asked to:
+
+- Read this document carefully discussing it with the Trustees for clarification if necessary
+
+The process for appointing the SIG committee is as follows:
+
+1. The founding steering committee was formed by volunteers from the 8 attendees at the initial meeting which agreed these terms of reference.
+2. Once established the SIG will manage it’s own appointments, launching a call for volunteers when required. 
+
+The open call is looking to fill these roles typically: (*chair, deputy-chair, secretary, finance*)
+
+## Onboarding of Committee members
+
+The Society asks that the Steering Group ensure that an initial onboarding meeting takes place to cover the following (but not limited to) activities:
+
+- Introductions and icebreaker
+- Circulate this document and ask all to read it carefully
+- Ask each member to take the free (mandatory) online EDI training on https://elearning.acas.org.uk/
+    - Bullying and harassment
+    - Discipline and grievance
+    - Equality, diversity and inclusion
+- Read the [Organisation Code of Conduct](https://society-rse.org/about/policies/code-of-conduct/)
```

Comment:
Not 100% sure whether this is the correct CoC (its the events one), or whether it should be the trustees/meetings one.